lib/appcanary/configuration.rb

```diff
@@ -0,0 +1,51 @@
+require "yaml"
+
+module Appcanary
+  APPCANARY_DEFAULT_BASE_URI = "https://appcanary.com/api/v3/monitors"
+
+  class Configuration
+    attr_accessor :base_uri, :api_token, :monitor_name
+
+    def initialize
+      self.base_uri = APPCANARY_DEFAULT_BASE_URI
+    end
+  end
+
+  class ConfigurationError < RuntimeError
+  end
+
+  class << self
```

Comment:
Is it ok to do this everywhere to get a usable module.method API style, or is there a better (or perhaps just existing) idiom I'm missing?